testing/trino-testing-containers/src/main/java/io/trino/testing/containers/TestContainers.java

```diff
@@ -30,6 +34,9 @@
     // TESTCONTAINERS_REUSE_ENABLE is an environment variable defined in testcontainers library.
     private static final boolean TESTCONTAINERS_REUSE_ENABLE = parseBoolean(getenv("TESTCONTAINERS_REUSE_ENABLE"));
 
+    // flag to understand is code running in CI environment or not
+    private static final boolean TESTCONTAINERS_IS_CI_ENV = parseBoolean(getenv("CONTINUOUS_INTEGRATION"));
```

Comment:
In other places we detect CI env by testing CONTINUOUS_INTEGRATION env var existence, without inspecting it's value